backend/core/src/listings/dto/listing.dto.ts

```diff
@@ -838,4 +861,5 @@ export const filterTypeToFieldMap: Record<keyof typeof ListingFilterKeys, string
   name: "listings.name",
   neighborhood: "property.neighborhood",
   bedrooms: "unitTypeRef.num_bedrooms",
+  leasingAgents: "leasingAgents.id",
```

Comment:
Also @seanmalbert this needed to be added to the ListingFilterKeysEnum here:
https://github.com/bloom-housing/bloom/blob/26a54444188e8c3c369a981e3b970ad419dff8f3/backend/core/src/listings/types/listing-filter-keys-enum.ts

Reply:
nvm it was! deleting this comment.